src/Microsoft.ML.Core/Data/IDataView.cs

```diff
@@ -161,8 +159,25 @@ public interface IRow : ISchematized, ICounted
         /// <summary>
         /// Returns a value getter delegate to fetch the given column value from the row.
         /// This throws if the column is not active in this row, or if the type
-        /// <typeparamref name="TValue"/> differs from this row's schema's
-        /// <see cref="ISchema.GetColumnType(int)"/> on <paramref name="col"/>.
+        /// <typeparamref name="TValue"/> differs from this column's type.
+        /// </summary>
+        ValueGetter<TValue> GetGetter<TValue>(int col);
+    }
+
+    /// <summary>
+    /// A logical row. Every value of every column is retrievable, and immutable.
+    /// </summary>
+    public interface IStandaloneRow : ISchematized
```

Comment:
>IStandaloneRow [](start = 21, length = 14)

significant change 2

Reply:
Actually not any more

---
In reply to: [224181114](https://github.com/dotnet/machinelearning/pull/1167#discussion_r224181114) [](ancestors = 224181114)